db/DB.js

```diff
@@ -219,6 +221,10 @@ const DB = {
       .del();
   },
 
+  async deleteSubscriberByFxAUID(fxaUID) {
+    await knex("subscribers").where("fxa_uid", fxaUID).del();
```

Comment:
I think this is OK. I was a bit confused since the `where` clause up on L172 above has `.where("id", "=", subscriber.id)`, but this doesn't have an "=" operator. But it looks like the docs imply that it is optional: https://knexjs.org/#Builder-wheres

The only other interesting thing from the docs was that an error may get thrown if `fxaUID` is `undefined` above:

> **Important:** Supplying knex with an `undefined` value to any of the where functions will cause knex to throw an error during sql compilation. This is both for yours and our sake. Knex cannot know what to do with `undefined` values in a where clause, and generally it would be a programmatic error to supply one to begin with. The error will throw a message containing the type of query and the compiled query-string.

Not sure if we want to add a bit more footgun protections by wrapping in a `try..catch` block, or if bubbling errors are caught higher up the flow and I'm just not seeing everything.

Reply:
Yeah, good idea. I'll add that tomorrow before merge.

Reply:
After looking at the flow, I think it's okay to let the error throw from here so that it can record into Sentry with as much of the stack as possible. These requests are never sent by humans, so we don't need to return a human-friendly error message here.